src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -160,15 +160,15 @@ public static String getMockAds() {
   }
 
   public static Boolean getForceHttps() {
-    return "true".equals(getProp("forceHttps"));
+    return "true".equals(getProp("forceHttps").toLowerCase());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1157)